source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -15,54 +15,73 @@ Bulk Operations
 Overview
 --------
 
-In this guide, you can learn how to use **bulk operations**.
+In this guide, you can learn how to use {+driver-short+} to
+perform **bulk operations**. Bulk operations reduce the number
+of calls to the server by performing multiple write operations
+in a single method.
 
-Bulk operations perform a large number of write operations. Instead of
-making a call for each operation to the database, bulk operations
-perform multiple operations with one call to the database.
+The ``Collection`` and ``Client`` classes both provide a ``BulkWrite()``
+method. When calling ``BulkWrite()`` on a ``Collection`` instance, you can
+perform multiple write operations on a single collection. When calling
+``BulkWrite()`` on a ``Client`` instance, you can perform bulk writes across
+multiple namespaces. In MongoDB, a namespace consists of the database name and the collection
+name in the format ``<database>.<collection>``. 
```

Comment:
S: in the bulk write method the namespaces are not given in that format so I recommend removing to avoid confusion
```suggestion
method. You can use the ``Collection.BulkWrite()`` method to
perform multiple write operations on a single collection. When calling
the ``Collection.BulkWrite()`` method, you can perform bulk writes across
multiple namespaces. In MongoDB, a namespace consists of a database name and a collection name. 
```